docs/internal/release-procedure.md

```diff
@@ -97,3 +107,28 @@ https://www.arduino.cc/en/software#future-version-of-the-arduino-ide
   >
   > To see the details, you can take a look at the [Changelog](https://github.com/arduino/arduino-ide/releases/tag/2.0.0-beta.12)
   > If you want to post about it on social media and you need more details feel free to ask us on #team_tooling! :wink:
+
+## Operations
+
+The following are detailed descriptions of operations performed during the release process:
+
+<a id="update-version-metadata"></a>
+
+### ⚙ Update version metadata of packages
+
+You need to **set the new version in all the `package.json` files** across the app (`./package.json`, `./arduino-ide-extension/package.json`, and `./electron-app/package.json`), create a PR, and merge it on the `main` branch.
+
+To do so, you can make use of the `update:version` script.
+
+For example, if you want to update the version to `<YOUR_VERSION>`, you should run the following commands:
+
+```text
+git checkout main
+git pull
+git checkout -b version-<YOUR_VERSION>
+yarn update:version <YOUR_VERSION>
+git commit -am <YOUR_VERSION>
+git push origin version-<YOUR_VERSION>
```

Comment:
Assuming you have set the remote to `origin`.

Reply:
This pull request only moved the existing content from one place in the document to another.